modules/platform/nuxeo-platform-importer/nuxeo-importer-stream/src/test/java/org/nuxeo/importer/stream/tests/importer/TestAutomation.java

```diff
@@ -62,7 +63,7 @@
     AutomationService automationService;
 
     @Rule
-    public TemporaryFolder folder = new TemporaryFolder();
+    public TemporaryFolder folder = new TemporaryFolder(new File("target"));
```

Comment:
I suggest you use `FeaturesRunner.getBuildDirectory()` istead of the hardcoded `target` to make sure that the multi-environment tests are completely isolated.
See https://jira.nuxeo.com/browse/NXP-28579 and https://github.com/nuxeo/nuxeo/commit/7d7a2f91b7f.

Reply:
Thanks, I will do that for modules that uses FeaturesRunner (i.e. not for the nuxeo-stream lib)